src/applications/mhv-landing-page/components/LandingPage.jsx

```diff
@@ -33,7 +52,7 @@ const LandingPage = ({ data = {} }) => {
             <Welcome name={name} />
           </>
         )}
-        {showCards ? <CardLayout data={cards} /> : <NoHealthAlert />}
+        {showCards ? <CardLayout data={cards} /> : noCardsDisplay}
```

Comment:
I could go either way, having the original format just kinda shuffles the ternary logic into a variable which only helps a little, and having the chained && all clumped together puts everything in one place but seems a bit harder to read at first. 🤷🏼

Reply:
To me, both approaches are a little confusing. I thought by making one "decision" early (what might appear instead of cards), that I could avoid having many conditionals in the render body.

I found the names of the selectors confusing, since names like `hasHealthData` and `isLOA1` sound like they'd be the boolean value output of a selector… 

I'd almost find something like an `if…else` clearer, e.g.:

```js

  const Body = () => {
    if (isUnverified) {
    // Unverified account
      return (
        <IdentityNotVerified
          headline={unVerifiedHeadline}
          showHelpContent={false}
        />
      );
    } else if (!hasHealth) {
      // No health data
      return <NoHealthAlert />;
    } else {
      // Show MHV
      return <CardLayout data={cards} />;
    }
  };
```

Where `<Body />` becomes part of the JSX in the top-level component. But I may be in the minority on that one